elyra/cli/pipeline_app.py

```diff
@@ -526,15 +538,29 @@ def describe(json_option, pipeline_path):
     describe_dict["type"] = primary_pipeline.type
     describe_dict["version"] = primary_pipeline.version
     describe_dict["nodes"] = len(primary_pipeline.nodes)
+    describe_dict["scripts"] = set()
+    describe_dict["notebooks"] = set()
     describe_dict["file_dependencies"] = set()
     describe_dict["component_dependencies"] = set()
+    describe_dict["mounted_volumes"] = set()
     for node in primary_pipeline.nodes:
         # collect information about file dependencies
         for dependency in node.get_component_parameter("dependencies", []):
             describe_dict["file_dependencies"].add(f"{dependency}")
         # collect information about component dependencies
         if node.component_source is not None:
             describe_dict["component_dependencies"].add(node.component_source)
+        # collect information of mounted volumes
+        for mounted_volume in node.get_component_parameter("mounted_volumes", []):
+            describe_dict["mounted_volumes"].add(f"{mounted_volume}")
+            # mounted_volume_set.add(f"{mounted_volume}")
+        # collect notebook / script name when pipeline is generic
+        if describe_dict["type"] is None:
+            temp_value = node.get_component_parameter("filename", [])
+            if temp_value.split(".")[1].strip() == pipeline_constants.NOTEBOOK_EXT:
```

Comment:
`temp_value.split(".")[1].strip()` does not yield the expected value if the filename contains more than one `.`. It's safer to use [pathlib's `Path.suffix`](https://docs.python.org/3/library/pathlib.html), e.g. `Path(temp_value).suffix`.